tests/Integration/Behaviour/Features/Context/OrderFeatureContext.php

```diff
@@ -188,11 +190,66 @@ public function checkOrderDiscount($position, $discountTaxIncluded, $discountTax
         }
     }
 
+    /**
+     * @Then created order should have :quantity products in total
+     */
+    public function createdOrderShouldHaveProductsInTotal($quantity)
```

Comment:
```suggestion
    public function assertOrderProductsTotal($quantity)
```

And later we can improve it into a generic `assertOrderProperties($orderId, TableNode $table)` function to which we pass a `$data` array and we match the different properties.